### PR TITLE
[feat] Add support for animated image formats

### PR DESCRIPTION
**Pull request by devfox-se**

### CODE REVIEW

**Comment by devfox-se** [comment]
Covers: #1708

File: aim/sdk/objects/image.py [hunk 1]
```diff
@@ -136,27 +131,51 @@ def _from_pil_image(self, pil_image: PILImage.Image, params):
...
-
-            alpha = pil_image.convert('RGBA').split()[-1]  # Get only alpha
-            background = PILImage.new('RGBA', pil_image.size, (255, 255, 255, 255))
-            background.paste(pil_image, mask=alpha)
-            pil_image = background.convert('RGB')
-
-            # Retry
-            pil_image.save(img_container, **params)
+        if not params['format']:
+            params['format'] = pil_image.format or 'PNG'
```
Comment (alberttorosyan) [inline aim/sdk/objects/image.py:135]: why switch to uppercase? the 'format' is [mostly] used internally, what's the motivation of this change?

**Review by alberttorosyan — COMMENTED** [review]
please update CHANGELOG as well

File: aim/sdk/objects/image.py (hunk 1 above)
Comment (devfox-se) [reply aim/sdk/objects/image.py:135]: I wanted to alight with PIL standard but since its only for internal I'll revert it

**Comment by devfox-se** [comment]
@alberttorosyan CHANGELOG was already updated

File: aim/sdk/objects/image.py [hunk 2]
```diff
@@ -174,6 +193,8 @@ def _from_numpy_array(self, array: np.ndarray, params):
         if array.ndim not in {2, 3}:
             raise ValueError('Cannot convert to aim.Image. array must have 2/3-D shape.')
 
+        if array.dtype == np.float:
+            array = np.asarray(array, np.uint8)
```
Comment (mihran113) [inline aim/sdk/objects/image.py:197]: why is this conversion needed?

File: aim/sdk/objects/image.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import logging
 import os.path
 
-from PIL import Image as PILImage
+from PIL import Image as PILImage, ImageSequence
```
Comment (mihran113) [inline aim/sdk/objects/image.py:4]: can we import `ImageSequence` as `PILImageSequence`? I got confused for a moment, when looking at the code I thought it was `aim.Images` 😆

**Review by mihran113 — APPROVED** [review]
left a minor suggestion and question there, other than that everything looks fine

File: aim/sdk/objects/image.py (hunk 3 above)
Comment (devfox-se) [reply aim/sdk/objects/image.py:4]: yeah nice recommendation:)

File: aim/sdk/objects/image.py (hunk 2 above)
Comment (devfox-se) [reply aim/sdk/objects/image.py:197]: but i think ill revert it :\
